### DISCUSSION TITLE
Question of crop and preprocess quality

### DISCUSSION THREAD

**Discussion by qiudashuai1** [CoBrALab/RABIES#392] [General]
Hi, I have preprocessed my data using the following command :
sudo docker run -it --rm --user $(id -u) \
-v $PWD/input_BIDS:/input_BIDS:ro \
-v $PWD/preprocess_outputs:/preprocess_outputs/ \
ghcr.io/cobralab/rabies:latest -p MultiProc preprocess /input_BIDS/ /preprocess_outputs/ --apply_STC --TR 2 --bold_autobox --commonspace_reg  masking=true,brain_extraction=false,template_registration=SyN,fast_commonspace=false
My data contains excessive neck tissue, so I used the --bold_autobox option to crop the data. However, it appears to be the same as the uncropped data. Is there have other recommended method to crop the brain? Additionally, the preprocessed data has some artifacts. I have limited knowledge about mouse fMRI. Is this preprocessing quality correct?
This is my origin data
![image](https://github.com/user-attachments/assets/835767ee-2e40-4123-bbe0-2ab7d3765cb7)
This is the preprocessed data
![image](https://github.com/user-attachments/assets/4f5ebc3b-d79f-4142-9a74-2f6a402fa899)
And the QC image
![image](https://github.com/user-attachments/assets/95f58df3-fe6b-49b8-9859-af2f24b77b5f)
[rabies_preprocess.log](https://github.com/user-attachments/files/17532900/rabies_preprocess.log)
![sub-con1_task-rest_bold_inho_cor](https://github.com/user-attachments/assets/c54e0ff3-42b7-47ce-be41-5694b356b439)
[sub-con1_task-rest_bold_RAS_autobox_inho_cor.nii.gz](https://github.com/user-attachments/files/17532901/sub-con1_task-rest_bold_RAS_autobox_inho_cor.nii.gz)

**Comment by Gab-D-G** [comment]
Hi, 
Regarding cropping, the only automated cropping tool in RABIES is the autobox option, which may fail in some subjects. You may try to manual crop data a priori, but maybe preprocessing will work fine regardless if don't use cropping.

The QC report you are showing looks fine for inhomogeneity correction. You will also have to inspect the registration reports.

**Reply by qiudashuai1** [reply]
This is my resistration figure, we do not scan cerebellum in fMRI.
![sub-con1_task-rest_bold_registration](https://github.com/user-attachments/assets/feae1725-57a4-499a-9319-0531669fb5c9)

**Reply by Gab-D-G** [reply]
This looks good for the most part, except the back of the brain where the cerebellum is a bit less accurate, so I recommend being careful when interpreting those regions.
